lib/module.js

```diff
@@ -250,7 +250,7 @@ Module._resolveLookupPaths = function(request, parent) {
 
   // make sure require('./path') and require('path') get distinct ids, even
   // when called from the toplevel js file
-  if (parentIdPath === '.' && id.indexOf('/') === -1) {
+  if (parentIdPath === '.' && id.includes('/')) {
```

Comment:
is this correct??
previous code indicated `id.indexOf('/') === -1`, I think `!id.includes('/')` is correct.
```js
if (parentIdPath === '.' && !id.includes('/')) {
}
```

But I don't understand why test are passed...